src/SeleniumLibrary/utils/webdrivercache.py

```diff
@@ -17,33 +17,33 @@
 from robot.utils import ConnectionCache
 
 
-class BrowserCache(ConnectionCache):
+class WebDriverCache(ConnectionCache):
 
     def __init__(self):
-        ConnectionCache.__init__(self, no_current_msg='No current browser')
+        ConnectionCache.__init__(self, no_current_msg='No current driver')
         self._closed = set()
 
     @property
-    def browsers(self):
+    def drivers(self):
         return self._connections
 
-    def get_open_browsers(self):
-        open_browsers = []
-        for browser in self._connections:
-            if browser not in self._closed:
-                open_browsers.append(browser)
-        return open_browsers
+    def get_open_drivers(self):
```

Comment:
Open driver sounds a bit odd. Should this perhaps be `get_active_browsers()`? Or perhaps property `active_browsers`?

Reply:
Did go with `active_drivers`